### PR TITLE
ICP-12050, ICP-12327: Add MultiChannelCmdEncap support to Z-Wave Metering Dimmer

### PR DESCRIPTION
**Pull request by dkirker**
The Aeotec Nano Dimmer seems to send us these messages, so we should make sure that we handle them.

### CODE REVIEW

**Review by kianooshkarami — DISMISSED** [review]

File: devicetypes/smartthings/zwave-metering-dimmer.src/zwave-metering-dimmer.groovy [hunk 1]
```diff
@@ -295,6 +295,25 @@ def zwaveEvent(physicalgraph.zwave.commands.crc16encapv1.Crc16Encap cmd) {
 	}
 }
 
+def zwaveEvent(physicalgraph.zwave.commands.multichannelv3.MultiChannelCmdEncap cmd) {
+	if (cmd.commandClass == 0x6C && cmd.parameter.size >= 4) { // Supervision encapsulated Message
```
Comment (greens) [inline devicetypes/smartthings/zwave-metering-dimmer.src/zwave-metering-dimmer.groovy:299]: @kianooshST is this still necessary? I thought this was a temp fix until it was handled in hubcore?

Comment (dkirker) [reply devicetypes/smartthings/zwave-metering-dimmer.src/zwave-metering-dimmer.groovy:299]: @greens @kianooshST Does hubcore do anything special with unwrapping the packet that this DTH isn't accounting for? (I believe that this DTH is supposed to be local, but I guess not?)

Comment (dkirker) [reply devicetypes/smartthings/zwave-metering-dimmer.src/zwave-metering-dimmer.groovy:299]: @kianooshkarami ping For some reason the device I was testing needed this (I don't recall removing the flags for local execution).

I am probably going to remove this block of code (there is another line that I think I left out of this PR -- which would explain why SRPOL still had this not work).

Comment (kianooshkarami) [reply devicetypes/smartthings/zwave-metering-dimmer.src/zwave-metering-dimmer.groovy:299]: Yes sorry. Please go ahead and remove this bit of code.

Comment (dkirker) [reply devicetypes/smartthings/zwave-metering-dimmer.src/zwave-metering-dimmer.groovy:299]: @kianooshkarami Thanks! Ok, I will have to see if the device I have still doesn't work after removing this. It didn't a month ago.

Comment (kianooshkarami) [reply devicetypes/smartthings/zwave-metering-dimmer.src/zwave-metering-dimmer.groovy:299]: it does not hurt keeping it

Comment (dkirker) [reply devicetypes/smartthings/zwave-metering-dimmer.src/zwave-metering-dimmer.groovy:299]: @kianooshkarami Ok! I think I'll keep it, unless @greens or anyone else have any strong opinions.

**Review by greens — APPROVED** [review]

**Pull request closed without merging**